src/Compilers/CSharp/Desktop/CSharpCodeAnalysis.Desktop.csproj

```diff
@@ -38,7 +38,6 @@
     </ProjectReference>
   </ItemGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DefineConstants>TRACE;DEBUG</DefineConstants>
```

Comment:
So where did `TRACE;DEBUG` move to? I expected to see them added to one of the .targets files... did I just miss that? Or are these already defined in Debug builds and the deleted line is extraneous?